scripts/check-frontend-dev.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env sh
+
+# Exit early if running in CI environment
+if [ -n "$CI" ] || [ -n "$GITHUB_ACTIONS" ] || [ -n "$IGNORE_NODE_VERSION_CHECK" ]; then
+  exit 0
+fi
+
+
+# Colors for prettier output
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+CYAN='\033[0;36m'
+NC='\033[0m' # No Color
+BOLD='\033[1m'
+
+
+REQUIRED_VERSION=$(cat .nvmrc | sed 's/v//')
```

Comment:
```suggestion
if [ -f .nvmrc ]; then
   echo "NVMRC file not found. run "git checkout main -- .nvmrc"
fi
REQUIRED_VERSION=$(cat .nvmrc | sed 's/v//')
```

note: this might not be syntax correct bash

Reply:
someone might remove it. just better ux.